charts/controller/templates/deployment.yaml

```diff
@@ -65,3 +67,23 @@ spec:
               port: 7488
             initialDelaySeconds: 15
             periodSeconds: 20
+{{ if (eq .Values.controller.serverMode "TLS") }}
+          volumeMounts:
+          - mountPath: /var/controller-tls-secrets/tls.key
+            subPath: tls.key
+            name: tls-key
+            readOnly: true
+          - mountPath: /var/controller-tls-secrets/tls.crt
+            subPath: tls.crt
+            name: tls-cert
+            readOnly: true
+{{ end }}
+{{ if (eq .Values.controller.serverMode "TLS") }}
+      volumes:
+      - name: tls-key
+        secret:
+          secretName: rookout-tls-key
```

Comment:
use controller.tlsKeySecretName and controller.tlsCertificateConfigmapName

Reply:
I wanted to hardcode it so the customer won't need to pass it to the template, just define them in k8s

Reply:
so what's the usage of the variables `controller.tlsKeySecretName` and `controller.tlsCertificateConfigmapName` that you added?

Reply:
removed ... :)